docs/usage/cli.rst

```diff
@@ -185,6 +185,19 @@ The ``routes`` command displays a tree view of the routing table.
 
    litestar routes
 
+Options
+~~~~~~~
+
++-----------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Flag            | Description                                                                                                                                           |
++=================+=======================================================================================================================================================+
+| ``--schema``    | Include default auto generated openAPI schema routes                                                                                                  |
++-----------------+ ------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``--exclude``   | Exclude endpoints from query with given regex patterns. multiple excludes allowed. e.g.  litestar routes  --schema --exclude=routes/.* --exclude=[]   |
++-----------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Unsure why this isnt rendering properly

Reply:
<img width="918" alt="image" src="https://github.com/litestar-org/litestar/assets/45884264/ae5867f8-3cf7-4ecc-90b0-c9de7094cf67">

Reply:
i think it was missing a - and the | wasn't aligned with the other ones. it looks good now.
![image](https://github.com/litestar-org/litestar/assets/7662358/7c4b8207-c0ff-41b4-a975-7f72f911df05)